html-aam/scripts/html-aam-accname-utils.js

```diff
@@ -0,0 +1,338 @@
+const HtmlAamAccnameUtils = {
```

Comment:
Thanks for making this super configurable!

Reply:
Sure thing! Am hoping for maximum coverage with minimal maintenance. 🤞🏻